assets/js/components/dashboard-sharing/ModuleRecoveryAlert/index.stories.js

```diff
@@ -157,6 +153,7 @@ MultipleRecoverableModuleNoAccess.args = {
 	},
 };
 MultipleRecoverableModuleNoAccess.scenario = {
+	// eslint-disable-next-line sitekit/no-storybook-scenario-label
 	label: 'Global/ModuleRecoveryAlert/Multiple Recoverable Modules (no access)',
```

Comment:
For the issue where this is updated, this component isn't Global, but specific to the dashboard 😇

Reply:
Added a reference to this comment in the ticket to resolve this https://github.com/google/site-kit-wp/issues/9103#issuecomment-2713502737.